lib/ts/recipe/dashboard/recipe.ts

```diff
@@ -127,6 +141,12 @@ export default class Recipe extends RecipeModule {
                 disabled: false,
                 method: "get",
             },
+            {
+                id: DASHBOARD_API,
+                pathWithoutApiBasePath: new NormalisedURLPath(getApiPathWithDashboardBase("/roles")),
```

Comment:
why are you not following the pattern of other objects in this array? They all use constants, so you should too. Also, which API is `/roles`?